lib/experimental/auto-inserting-blocks.php

```diff
@@ -0,0 +1,246 @@
+<?php
+/**
+ * Auto-inserting blocks.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Return a function that auto-inserts blocks relative to a given block.
```

Comment:
Would be really handy to have an alert up here that the function this returns mutates the blocks it's passed.

Reply:
Good point. Addressed in 4c312b8037, where I've also added some more explanation of how this function works.